docs/release-process.md

```diff
@@ -385,7 +385,7 @@ explained at the end of this document in
     Example:
     https://github.com/php/php-src/commit/a63c99ba624cff86292ffde97089428e68c6fc10
 
-    Push the new branch and the commit just added to master.
+    Push the new branch, and the commit just added to master.
```

Comment:
Ditto

Reply:
Thanks! Reading it a second time makes it better without the comma.